frontend/src/components/WofFooter.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+    <footer>
+        <img class="footer__ul-img" src="../assets/logo-ul-eng.png">
+        <div class="footer__authors">
+            <h4>Authors:</h4>
+            <span>Karol Kozłowski</span>
+            <span>Konrad Pagacz</span>
+            <span>Robert Szczurkowski</span>
+            <span>Magdalena Wiechecka</span>
+            <span>Kacper Fryczak</span>
+        </div>
+    </footer>
+</template>
+
+<script>
+export default {
+    name: 'WofFooter',
+}
+</script>
+
+<style lang="less">
+@import './common.less';
+
+footer {
+    display: flex;
+    background-color: @primary-theme-color;
+    padding: 10px;
+
+    .footer__ul-img {
+        width: 14em;
+    }
+
+    .footer__authors {
+        display: flex;
+        flex-direction: column;
+        color: @white;
```

Comment:
Is there a point of renaming the color definitions to something like main text secondary text etc or does it go against the usual frontend practice? I imagine if we ever implement a different theme for the site having a variable named @white actually mean something different might be confusing. I am happy to do it in another task if you think it's worth it.

Reply:
Name 'white' is ok because it's OUR definition of white color.

Reply:
Indeed, but at some point it might stop being white and start being blue (if we ever bother to implement another theme for it)

Reply:
Ok, i changed the names